src/app/data-planes/data/index.ts

```diff
@@ -86,6 +85,8 @@ const DataplaneNetworking = {
     const { inbound, outbound, ...rest } = networking
 
     const inbounds = Array.isArray(inbound) ? inbound : []
+    // TODO(jc): Confirm whether we still ever get outbounds, from what I understand
+    // we no longer get this property
```

Comment:
we can if we have Universal DPP without transparent proxying

Reply:
Ah perfect! Thanks Jakub! Let me replace this with an explanatory comment